src/components/menu/menu.test.js

```diff
@@ -1400,4 +1449,436 @@ context("Testing Menu component", () => {
         });
     });
   });
+
+  describe("Accessibility tests for Menu component", () => {
+    it("should pass accessibility tests for Menu default", () => {
+      CypressMountWithProviders(<MenuComponent />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should pass accessibility tests for Menu expanded", () => {
+      CypressMountWithProviders(<MenuComponent />);
+
+      submenu()
+        .eq(positionOfElement("first"), div)
+        .trigger("mouseover")
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          cy.checkAccessibility();
+        });
+    });
+
+    it.each(["default", "large"])(
+      "should pass accessibility tests for Menu when divider size is %s",
+      (divider) => {
+        CypressMountWithProviders(<MenuComponent size={divider} />);
+
+        submenu()
+          .eq(positionOfElement("first"), div)
+          .trigger("mouseover")
+          .then(() => {
+            // eslint-disable-next-line no-unused-expressions
+            cy.checkAccessibility();
+          });
+      }
+    );
+
+    it("should pass accessibility tests for Menu when search component is focused", () => {
+      CypressMountWithProviders(<MenuComponentSearch />);
+
+      pressTABKey(1);
+      cy.wait(50);
+      cy.focused().trigger("keydown", keyCode("Enter"));
+      cy.wait(50);
+      pressTABKey(0);
+      cy.wait(50);
+      cy.focused()
+        .trigger("keydown", keyCode("downarrow"))
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          cy.checkAccessibility();
+        });
+    });
+
+    it("should pass accessibility tests for Menu when a menu item has a long label", () => {
```

Comment:
Is this test name transposed with the following one starting on line 1529 possibly? It looks like the submenu name is long in this one and vice versa.